src/Laravel/Validation/Rules/Float32Rule.php

```
@@ -0,0 +1,45 @@
+ * Validation rule for Float32 values
+ */
+final class Float32Rule implements Rule
+{
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param string $attribute
+     * @param mixed $value
+     * @return bool
+     */
+    public function passes($attribute, $value): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)